lib/Swagger/src/Discussion/Api/SitesApi.php

```diff
@@ -149,16 +144,13 @@ public function createSiteWithHttpInfo($site, $x_wikia_schwartz = null)
         }
         $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array('application/json'));
 
-        
         // header params
         if ($x_wikia_schwartz !== null) {
             $headerParams['X-Wikia-Schwartz'] = $this->apiClient->getSerializer()->toHeaderValue($x_wikia_schwartz);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "X-Wikia-Schwartz" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)